charts/caluma/Chart.yaml

```
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: caluma
 description: A collaborative form editing service
-version: 0.10.2
-appVersion: 7.7.0
+version: 0.10.3
```

When you upgrade the appVersion over multiple major releases, then you should definitely also increase the minor version of the chart.
```suggestion
version: 0.11.0
```